pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -720,7 +721,7 @@ func (r *ReconcileInfinispan) statefulSetForInfinispan(m *infinispanv1.Infinispa
 		}
 
 		pvc := &corev1.PersistentVolumeClaim{ObjectMeta: metav1.ObjectMeta{
-			Name:      m.Name,
+			Name:      DataMountVolume,
```

Comment:
What happens if you have two Infinispan CRs in the same namespace? Won't that result in two PVCs with the same name?

Reply:
I will check again, CI did the same, but seems to be nothing happens
PVC claim template name is <volume name>-<pod name> that's unique